bin/release

```diff
@@ -111,31 +111,51 @@ function pushTag(SymfonyStyle $ui, string $directory, string $branch, $tag): voi
 
         $ui->section('Configuration');
 
-        if ('y' !== \strtolower($ui->ask(\sprintf('Create tags for branch "%s"?', $branch), 'y'))) {
+        if ('y' !== \strtolower($ui->ask(\sprintf('Create new release for branch "%s"?', $branch), 'y'))) {
             $ui->text('Canceled by user.');
 
             return 0;
         }
 
-        $ui->text(\sprintf('Start tagging "%s" branch.', $branch));
+        $ui->text(\sprintf('Start creating release for "%s" branch.', $branch));
```

Comment:
same as above it create a tag not a release

Reply:
```suggestion
        $ui->text(\sprintf('Start tagging "%s" branch.', $branch));
```

Reply:
reverted this change 🙂